dapps/marketplace/src/components/MobileModal.js

```diff
@@ -231,7 +231,6 @@ require('react-styl')(`
       opacity: 1
       display: block
   .mobile-modal-light
-    touch-action: none
```

Comment:
@shahthepro can you remember why this was added? OK to merge?

Reply:
@nick Not that I can remember. But last time this issue popped up, @tomlinton had [opened this PR](https://github.com/OriginProtocol/origin/pull/2539) and it did work on Android as well as on iOS (with some scroll issues). So I think some other change would have caused this again and not this line. I'm not sure though.